pkg/controller/unifiedpushserver/unifiedpushserver.go

```diff
@@ -270,6 +272,16 @@ func newUnifiedPushServerDeploymentConfig(cr *pushv1alpha1.UnifiedPushServer) (*
 							Image:           cfg.UPSImageStreamName + ":" + cfg.UPSImageStreamTag,
 							ImagePullPolicy: corev1.PullAlways,
 							Env:             buildEnv(cr),
+							Resources: corev1.ResourceRequirements{
+								Limits: corev1.ResourceList{
+									"memory": resource.MustParse("2Gi"),
+									"cpu":    resource.MustParse("1"),
+								},
+								Requests: corev1.ResourceList{
+									"memory": resource.MustParse("512Mi"),
+									"cpu":    resource.MustParse("100m"),
```

Comment:
should we request a bit more CPU, maybe start with 500m?

Reply:
Sure thing